packages/plugin-doc-coverage/src/lib/config.unit.test.ts

```diff
@@ -0,0 +1,64 @@
+import { describe, expect, it } from 'vitest';
+import {
+  type DocCoveragePluginConfig,
+  docCoveragePluginConfigSchema,
+} from './config.js';
+
+describe('docCoveragePluginConfigSchema', () => {
+  it('accepts a documentation coverage configuration with all entities', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        language: 'typescript',
+        sourceGlob: 'src/**/*.{ts,tsx}',
+      } satisfies DocCoveragePluginConfig),
+    ).not.toThrow();
+  });
+
+  it('accepts minimal configuration with only language', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        language: 'javascript',
+      } satisfies DocCoveragePluginConfig),
+    ).not.toThrow();
+  });
+
+  it('accepts configuration without sourceGlob', () => {
+    const config = {
+      language: 'typescript',
+    } satisfies DocCoveragePluginConfig;
+    const parsed = docCoveragePluginConfigSchema.parse(config);
+
+    expect(parsed.sourceGlob).toBeUndefined();
+  });
```

Comment:
This is already tested here https://github.com/code-pushup/cli/pull/896/files#r1889385014 and can get removed.